lib/mayaUsd/resources/scripts/mayaUsdLibRegisterStrings.mel

```diff
@@ -44,4 +44,7 @@ global proc mayaUsdLibRegisterStrings()
     register("kMayaRefCreateNew", "Create New");
     register("kMayaRefAddToPrim", "Add Maya Reference to USD Prim");
     register("kMayaRefReference", "Reference");
+    register("kMayaRefMergeEdits", "Merge Maya Edits to USD");
+    register("kMayaRefDiscardEdits", "Discard Maya Edits");
+    register("kMayaRefDuplicateAsUsdData", "Duplicate As USD Data");
```

Comment:
These are plugin strings and should go here: https://github.com/Autodesk/maya-usd/blob/dev/plugin/adsk/scripts/mayaUSDRegisterStrings.mel